src/Apps/Order2/Routes/Checkout/Hooks/useLoadCheckout.ts

```diff
@@ -0,0 +1,104 @@
+import createLogger from "Utils/logger"
+import type { useLoadCheckout_order$key } from "__generated__/useLoadCheckout_order.graphql"
+import { every } from "lodash"
+import { useCallback, useEffect, useState } from "react"
+import { useFragment } from "react-relay"
+import { graphql } from "react-relay"
+
+const logger = createLogger("useLoadCheckout")
+
+const MINIMUM_LOADING_MS = 1000
+
+type CheckoutLoadingError = "missing_line_item_data"
```

Comment:
we can build this type out more in the future so that we can handle things going wrong at load time - or we can delete all of this code for now.